demo/config.yaml

```diff
@@ -1,10 +1,10 @@
 lcr-modules:
 
     _shared:
-        lcr-modules: "../"
-        lcr-scripts: "../../lcr-scripts"
+        lcr-modules: "/projects/rmorin/projects/tumour_contam/lcr-modules/"
```

Comment:
Restore all paths to default. If this branch is relatively up to date with Master the defaults should all work. Not sure why you changed these.

Reply:
Reverted back